src/pages/home/report/ReportActionItem.js

```diff
@@ -232,6 +234,9 @@ class ReportActionItem extends Component {
 
         const reactions = _.get(this.props, ['action', 'message', 0, 'reactions'], []);
         const hasReactions = reactions.length > 0;
+        const shouldDisplayThreadReplies = (this.props.action.childCommenterCount || 0) > 0;
```

Comment:
is the `|| 0` needed? `undefined > 0` will return false anyway.

If `|| 0` is needed this could be easier to read like so,
```
const shouldDisplayThreadReplies = lodashGet(this.props, 'action.childCommenterCount', 0) > 0;
```